src/components/VenueSelector.js

```diff
@@ -77,6 +81,8 @@ const VenueSelector = () => {
 		const fullAddressUpdated = venueInformation?.fullAddress ?? '';
 		const phoneNumberUpdated = venueInformation?.phoneNumber ?? '';
 		const websiteUpdated = venueInformation?.website ?? '';
+		const latitudeUpdated = venueInformation?.latitude ?? '0';
+		const longitudeUpdated = venueInformation?.longitude ?? '0';
```

Comment:
I'd like to suggest, going with the WordPress Geodata standard (https://github.com/GatherPress/gatherpress/issues/560) at this point and call the post_meta keys geo_latitude and geo_longitude.

Reply:
No problem. I broke some stuff this morning trying to do this. Let's plan to update it as part of ticket 560